evap/results/templates/results_index_evaluation.html

```diff
@@ -57,14 +57,14 @@
         </div>
     {% endif %}
     {% if evaluation.is_single_result %}
-        <div data-col="participants" class="text-center" data-order="100"><span class="fas fa-user"></span>&nbsp;{{ evaluation.single_result_rating_result.count_sum }}</div>
+        <div data-col="participants" class="text-center"><span class="fas fa-user"></span>&nbsp;{{ evaluation.single_result_rating_result.count_sum }}</div>
     {% else %}
         {% with num_participants=evaluation.num_participants num_voters=evaluation.num_voters %}
-            <div data-col="participants" data-order="{{ num_voters|percentage_value:num_participants|default:'0' }}">{% include 'progress_bar.html' with done=num_voters total=num_participants %}</div>
```

Comment:
As we don't know how to sort the participants column, I removed the attributes. However, `percentage_value` now seams unused. Should I remove?

Reply:
yes please